src/Quantity.elm

```diff
@@ -504,6 +515,21 @@ sqrt (Quantity value) =
     Quantity (Basics.sqrt value)
 
 
+{-| Cube a quantity with some `units`, resulting in a new quantity in
+`Cubed units.
+-}
+cubed : Quantity number units -> Quantity number (Cubed units)
+cubed (Quantity value) =
+    Quantity (value * value * value)
+
+
+{-| Take a quantity in `Cubed units` and return the cube root of that
+quantity in plain `units.
+-}
+cbrt : Quantity Float (Cubed units) -> Quantity Float units
+cbrt (Quantity value) =
+    Quantity (value ^ (1 /3))
```

Comment:
For negative values this will give `NaN`. (i.e. in elm `(-8) ^ (1/3)` is `NaN`.)

This may be the desired behaviour but at the very least this should be documented as people may expect this to work for them.

Reply:
Excellent point @harrysarson, great catch! Given that we don't have direct access to `Math.cbrt` from JavaScript, I think the implementation should probably be updated to

```elm
cbrt (Quantity value) =
    if value >= 0 then
        Quantity (value ^ (1 / 3))
    else
        Quantity -((-value) ^ (1 / 3))
```

Does that make sense?

Reply:
@harrysarson Thanks!